src/pepr/operator/crd/generated/package-v1alpha1.ts

```diff
@@ -438,13 +438,14 @@ export interface Sso {
    */
   description?: string;
   /**
-   * Whether the SSO client is enabled
+   * Labels to match pods to automatically protect with authservice. Leave empty to disable
```

Comment:
I think an empty selector of `{}` like: `enableAuthserviceSelector: {}` should enable Authservice for all Pods in a namespace, correct?

I'd like to include this in the docs and recommend it as a default. I'd rather people only write narrow selectors if they need to exclude some Pods.

And this should probably say "leave null to disable" instead of "empty", to disambiguate between `{}` which is actually all.

Reply:
`enableAuthserviceSelector: {}` should enable authservice for all pods in the namespace, but I am curious to hear why that would be your recommendation for the default? 

Does that lean into an assumption of there being a single pod/deployment in the namespace? If a user deployed an application with an internal API or database, how would we recommend they authenticate?

Reply:
Just that I'd encourage auth on all webservers that are reachable (in-cluster/SSRF and external) and don't already have auth. If you're careful to write good policies it's fine to exclude things, but it's a safe default to encourage I think.

I'd recommend docs something like:
> Use `enableAuthserviceSelector: {}` to add authservice to your app. If you need to exclude some Pods, edit it as needed and verify that the excluded Pods are not reachable externally or from other namespaces without authentication.

Doing auth on `{}` is also future proof to the app changing in the future and adding new Pods that don't match the old selector.

Reply:
(there's also the actual "zero trust" answer, but it only applies to custom developed apps: Encourage people to develop apps that don't access their backend unauthenticated. Have them pass the user's JWT from the frontend Pod to the backend/db Pod as auth. In this setup, you can reuse istio JWT authn/authz checking... but you actually don't want the CUSTOM authservice authz. Not sure if this is something we want to build support for, but it actually results in good design of custom apps which create a great audit log in isto access logs)